templates/core/index.html

```diff
@@ -42,37 +42,6 @@ <h2>What is Django Girls?</h2>
                 </div>
             </div>
 
```

Comment:
Referring to my comment about donors page being a bit hidden in the menu: maybe we could leave it there, but have a section on the main page that features our donor's page a bit, so people could discover it easier - I think it would be nice for our donors to have that featured for some time. So we would need a section with a sentence or two about campaign and how greateful we are to our donors, a nice picture and link to the donors page. 

If we feel fancy, we could even fetch a couple of donor's names and randomly feature them on the main page?